kore/src/main/scala/org/kframework/builtin/Sorts.scala

```diff
@@ -39,4 +39,7 @@ object Sorts {
   val GeneratedCounterCell = Sort("GeneratedCounterCell")
 
   val Id = Sort("Id")
+
+  val Z3Query  = Sort("Z3Query")
```

Comment:
these are not real sorts. Is there a reason why we need to encode these as fake K terms instead of just logging them directly?